components/BuyTicketSection.js

```diff
@@ -10,7 +10,7 @@ export default function BuyTicketSection() {
                     <p className="text-base p-0 font-medium">Tickets can be bought by following the link below.</p>
                     <div className="py-12">
                         <Link className="bg-[#ffe135] text-white font-bold py-3 px-10 m-2 border rounded-3xl" href={QUICKET_LINK}>
-                            Buy a Ticket
+                            Register
```

Comment:
@basiima does this mean we are doing away with `buy ticket` for `register` ?

Reply:
Yes as agreed in the meeting, until the payments issues are resolved

Reply:
But the link remains...doesnt that mean we need to entirely remove the link too?